README.md

```diff
@@ -14,8 +14,13 @@ OpenWrt provides a fully writable filesystem with optional package management. T
 For developers, OpenWrt provides a framework to build an application without having to create a complete firmware image and distribution around it.
 For users, this means the freedom of full customisation, allowing the use of an embedded device in ways the vendor never envisioned.
 
-Creator is MIPS based platform series from Imagination and Marduk is the latest board under that series. Pistachio is the name of Imagination's SoC based on MIPS.
-This guide helps as a quick start but for full details please see the [OpenWrt project wiki](http://wiki.openwrt.org/doc/start).
+Creator is a MIPS based developer program from Imagination Technologies. Creator Ci40 (Marduk) is the latest development board in the Creator program. Pistachio is the name of the Imagination Technologies SoC that Ci40 is uses as it's core.
+This guide helps as a quick start but for full details about OpenWrt please see the [OpenWrt project wiki](http://wiki.openwrt.org/doc/start).
+
+
+## Using this documentation
+
+This readme contains the basics of building and modifying OpenWrt for Ci40. *You will find much more documentation on the [Creator Documentation website](http://docs.creatordev.io/ci40).*
```

Comment:
The link here to docs.creatordev.io should directly take you to https://docs.creatordev.io/ci40/introduction/dev-guide/ as that would be for refering to specific use of Ci40 OpenWrt guide.